packages/babel-plugin-import-jsx-pragma/index.js

```diff
@@ -1,17 +1,20 @@
 /**
  * Default options for the plugin.
  *
- * @property {string}  scopeVariable Name of variable required to be in scope
- *                                   for use by the JSX pragma. For the default
- *                                   pragma of React.createElement, the React
- *                                   variable must be within scope.
- * @property {string}  source        The module from which the scope variable
- *                                   is to be imported when missing.
- * @property {boolean} isDefault     Whether the scopeVariable is the default
- *                                   import of the source module.
+ * @property {string}  scopeVariable     Name of variable required to be in scope
+ *                                       for use by the JSX pragma. For the default
+ *                                       pragma of React.createElement, the React
+ *                                       variable must be within scope.
+ * @property {string}  scopeVariableFrag Name of variable required to be in scope
+ *                                       for use by the Fragment pragma.
+ * @property {string}  source            The module from which the scope variable
+ *                                       is to be imported when missing.
+ * @property {boolean} isDefault         Whether the scopeVariable is the default
+ *                                       import of the source module.
  */
 const DEFAULT_OPTIONS = {
 	scopeVariable: 'React',
+	scopeVariableFrag: null,
```

Comment:
So a consumer _must_ opt in to this? Can there not be a default, like there is with `@babel/plugin-transform-react-jsx` ?

Is the expected behavior when this is `null` to not do anything when it encounters a fragment?

Reply:
I think `null` matches the default expectations across the board. When it's `null` and `<></>` Fragments are encountered, no named import is added. However, this is JSX so the `scopeVariable` _will_ be added.

If we consider these to be the defaults:

```js
const webpackConfig = { // …
plugins: [
[
	require.resolve( '@wordpress/babel-plugin-import-jsx-pragma' ),
	{
		scopeVariable: 'React',
		scopeVariableFrag: null,
		source: 'react',
		isDefault: true,
	},
],
[ require.resolve( '@babel/plugin-transform-react-jsx' ), {
	pragma: 'React.createElement',
	pragmaFrag: 'React.Fragment',
} ],
};
```

Note that `pragmaFrag`'s default is `React.Fragment`. With default arguments, `<>In a Fragment</>` will result in something like:

```js
import React from 'react';
React.createElement( React.Fragment, "In a Fragment" );
```

Reply:
Oh! That's quite tricky to follow, but it makes sense. Thanks for explaining.